src/modules/esl-media-query/core/esl-media-rule-list.ts

```diff
@@ -130,7 +130,8 @@ export class ESLMediaRuleList<T = any> extends SyntheticEventTarget {
   public static parseTuple(mask: string, values: string, parser: RulePayloadParser<any> = String): ESLMediaRuleList {
     const queries = mask.split('|');
     const valueList = values.split('|');
-    if (valueList.length !== queries.length) throw new Error('Value doesn\'t correspond to mask');
+      while (valueList.length < queries.length && valueList.length !== 0) valueList.push(valueList[valueList.length - 1]);
```

Comment:
cosmetic-cosmetic - extra space
```suggestion
    while (valueList.length < queries.length && valueList.length !== 0) valueList.push(valueList[valueList.length - 1]);
```